app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -323,6 +323,29 @@ class Contact extends React.Component {
                     {this.state.errors['secondary_telephone_type']}
                   </Form.Control.Feedback>
                 </Form.Group>
+                <Form.Group as={Col} controlId="primary_phone_type_warning_message">
+                  {this.state.current.patient.preferred_contact_method === 'SMS Texted Weblink' &&
```

Comment:
this logic and language came from Kelly in a group chat that Alex made with me and Kelly when he was clarifying some things

![Screen Shot 2020-09-24 at 4 01 27 PM](https://user-images.githubusercontent.com/9956561/94194104-38795980-fe7f-11ea-96fa-b2ed5f863e7b.png)

Reply:
Gotcha. No need to post the Slack messages (since this is a public repo), but this is all set then. Re-reading the ticket it makes sense since plain cells can receive texts, just can't use the weblinks.